kubernetes.go

```diff
@@ -445,6 +455,13 @@ type ClusterlintOwner struct {
 	Name string `json:"name"`
 }
 
+// AssociatedResources represents a cluster's associated resources
+type AssociatedResources struct {
```

Comment:
```suggestion
// KubernetesAssociatedResources represents a cluster's associated resources
type KubernetesAssociatedResources struct {
```

Can we namespace this? Other resources (e.g. Droplets) may also have something like this.